test/factorials/factorial-test.js

```diff
@@ -0,0 +1,43 @@
+const assert = require('chai').assert
+const factorialCalculator = require('../../lib/factorials/factorial.js')
+
+describe('factorialCalculator functionality', function() {
+  context('factorialCalculator function', function(){
+    it('it can calculate the factorial of a number', function(){
+      let number = 0;
```

Comment:
same here. you are trying quite a few inputs which is a good in theory, but not necessary. i suggest only testing the edge cases along with one "normal" case - probably test 0 and also any integer.